db/data.py

```diff
@@ -101,14 +145,32 @@ def delete_spot(spot_id):
         return NOT_FOUND
     return response
 
+def update_factors(spot_id, factorAvailability=None,
+                   factorNoiseLevel=None,
+                   factorTemperature=None,
+                   factorAmbiance=None):
+    spot_document = {}
+    if factorAvailability:
+        spot_document["factorAvailability"] = factorAvailability
+    if factorNoiseLevel:
+        spot_document["factorNoiseLevel"] = factorNoiseLevel
+    if factorTemperature:
+        spot_document["factorTemperature"] = factorTemperature
+    if factorAmbiance:
+        spot_document["factorAmbiance"] = factorAmbiance
+
+    response = dbc.update_spot(spot_id, spot_document)
+    return response if response is not None else NOT_FOUND
+
 
-def add_review(spotID, reviewTitle, reviewText, reviewRating):
+def add_review(spotID, reviewDate, reviewTitle, reviewText, reviewRating):
     """
     Return a dictionary of created review.
     """
     review_object = {
         "_id": dbc.generate_id(),
         "spotID": spotID,
+        "reviewDate": reviewDate,
```

Comment:
Are `reviewDate` and `reviewCreation` going to be used differently?

Reply:
I think reviewDate is a better name for it

Reply:
When I added `reviewCreation` and `reviewUpdate`, I was thinking of them as the timestamps for object creation and update. So I think `reviewDate` would be a little different since that's like whatever the date the user puts in as the visiting date?

Reply:
I think `reviewCreation` would make the most sense since when they create the review, it is posted on that date. Would you have time to fix this?

Reply:
I think keeping both works as well! If we keep both, then `reviewCreation` and `reviewUpdate` would remain as timestamps for the review while `reviewDate` would be like the visiting date that a user puts in. So a user would be allowed to post a review for a date in the past as opposed to having a generic review. But for now, I'll do the change you asked about.